external-crates/move/crates/move-compiler/src/shared/ide.rs

```diff
@@ -82,9 +82,9 @@ pub struct DotAutocompleteInfo {
 #[derive(Default, Debug, Clone)]
 pub struct AliasAutocompleteInfo {
     /// Numerical addresses that are valid autocompletes
-    pub addresses: BTreeSet<(Symbol, NumericalAddress)>,
+    pub addresses: BTreeMap<Symbol, NumericalAddress>,
```

Comment:
What is the motivation for this change?

Reply:
Faster look up on the symbol key. Not a big issue at all (and not a big gain :-) )  - can certainly revert if that's preferrable